lib/Munin/Node/Server.pm

```diff
@@ -216,6 +216,7 @@ sub _process_command_line {
 sub _get_commandline {
   my $self = shift;
 
+  ## no critic Variables::ProhibitPunctuationVars
```

Comment:
How about following the recommendation of `man Perl::Critic::Policy::Variables::ProhibitPunctuationVars` and using `English` and `$PROGRAM_NAME`?
(I am not aware of side-effects of `English`)

Reply:
done